crates/ruff/resources/test/fixtures/unformatted.py

```diff
@@ -1,3 +1,3 @@
 x = 1
-y=2
+y = 2
```

Comment:
the title of this file suggests maybe the formatting violation here might have been deliberate? ;)

Reply:
Thanks we should exclude these resources.. I added this in a later commit perhaps?

Reply:
https://github.com/astral-sh/ruff/pull/10698/commits/e5b10e53741e453355aa2c01773a67fc834825e2